src/middleware/datasetTaskList.middleware.js

```
@@ -113,14 +123,17 @@ const validateParams = validateQueryParams({
   })
 })
 
+/* eslint-disable-next-line no-return-assign */
+const zeroEntityCount = (req) => req.entityCount = { entity_count: 0 }
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)